ui-components/src/wasmJsMain/kotlin/org/jetbrains/kotlinconf/ui/InitCoil.kt

```diff
@@ -0,0 +1,38 @@
+package org.jetbrains.kotlinconf.ui
+
+import coil3.ImageLoader
+import coil3.PlatformContext
+import coil3.SingletonImageLoader
+import coil3.intercept.Interceptor
+import coil3.request.ImageResult
+import coil3.util.DebugLogger
+
+fun initCoil() {
+    SingletonImageLoader.setSafe {
+        ImageLoader.Builder(PlatformContext.INSTANCE)
+            .components {
+                add(SessionizeImageInterceptor())
+            }
+            .logger(DebugLogger())
+            .build()
+    }
+}
+
+private val sessionizeProxy = "https://sessionize-com.labs.jb.gg/"
+
+private class SessionizeImageInterceptor : Interceptor {
+    override suspend fun intercept(chain: Interceptor.Chain): ImageResult {
+        val originalRequest = chain.request
+        val data = originalRequest.data
+
+        val newChain = if (data is String && data.startsWith("https://sessionize.com/")) {
+            val newUri = data.replace("https://sessionize.com/", sessionizeProxy)
```

Comment:
please consider extracting URLs to resources or constants

Reply:
Oops, don't know how I didn't do that. Fixed :)